rest-service/setup.py

```diff
@@ -40,6 +40,7 @@
     'retrying',
     'pydantic',
     'distro',
+    'gunicorn',
```

Comment:
no no no, this isn't a dependency of the restservice itself, that's just how it happens to be deployed (but it doesn't need to be gunicorn)

more practically: we don't want to have to install gunicorn to run tests